### ISSUE TITLE
bug(YouTube): Videos take forever to load. Not playing

### ISSUE THREAD

**Issue by mogley25** [ReVanced/revanced-patches#2638]
### Bug description

When trying to play videos, some videos take forever to load. This started the few days ago.

Currently using recommended version.

Running only the following patches.

GmsCore support
Hide ads
Premium heading
Restore old video quality menu
Swipe controls
Video ads

![Screenshot_20240122-131035](https://github.com/ReVanced/revanced-patches/assets/43046761/b1ea23bc-8073-4e60-a809-ac3720322a71)

### Acknowledgements

- [X] This issue is not a duplicate of an existing bug report.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by oSumAtrIX** [comment]
Follow this and report back:

![image](https://github.com/ReVanced/revanced-patches/assets/13122796/2fe274be-717b-4867-a99e-2ba4c7dcbc5f)

**Comment by mogley25** [comment]
Alright will do this. Cant disable GMS core tho then I can't install the app. Im not root. Il disable one by 1 and figure out the problemsome patch

**Comment by mogley25** [comment]
Tried Lisos ideas. Still doesnt wanna work.

Also Ive disabled all ad blocking plugins and the issue is still present. This only started happening a few days ago.

**Comment by mogley25** [comment]
Yes it seems to be happening with the youtube app itself too but not as bad as revanced. Sorry about that.

**Comment by mogley25** [comment]
I'll try on another device as well to see if it's totally YouTube problem or if it's something with my phone

**Comment by oSumAtrIX** [comment]
Try using the unpatched app for a while and check if the problem is resolved through that. It is not impossible that the behaviour is induced by ReVanced into the unpatched app via server-side variables.

**Comment by rutvik788845** [comment]
Try not to use same account on 2 different devices on same revanced app

**Comment by Game-Warrior** [comment]
> [@mogley25](https://github.com/mogley25) also try turning off storyboard spoofing (in Misc settings), and see if that changes anything. A slow storyboard api call can cause delays in opening a new video.

This worked for me. I disabled video spoofing and YouTube premium downloads now work as expected.
